frontend/public/components/cluster-settings/global-config.tsx

```diff
@@ -32,15 +32,21 @@ const oauthMenuItems = _.map(addIDPItems, (label: string, id: string) => ({
 }));
 
 const ItemRow = ({ item }) => {
-  const resourceLink = resourcePathFromModel(item.model, item.name, item.namespace);
-  const apiExplorerLink = `/api-resource/cluster/${referenceForModel(item.model)}`;
-  const menuItems = [
-    editYAMLMenuItem(item.kind, resourceLink),
-    viewAPIExplorerMenuItem(item.kind, apiExplorerLink),
-    ...(item.kind === 'OAuth' ? oauthMenuItems : []),
-  ];
-  const description = getResourceDescription(item.model);
-
+  let resourceLink, menuItems, description;
+  if (item.isK8Item !== false) {
```

Comment:
This isn't the right place to have this logic. It would be better to pass in an item object with `link`, `actions`, and `description` already set. Then the logic here is trivial. The way to build those items should be moved up to a higher level that already knows what kind of config link it's working with.

Reply:
fixed